jmix-dynattr/dynattr/src/main/java/io/jmix/dynattr/model/JavaFieldNameSupportedSymbols.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.dynattr.model;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+
+import java.lang.annotation.*;
+
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Constraint(validatedBy = JavaFieldNameSupportedSymbolsValidator.class)
+public @interface JavaFieldNameSupportedSymbols {
```

Comment:
You create an annotation for a specific attribute, so let's name it respectively. Now, it seems that I can apply it to any attribute and define any supported symbols by regexp, which is not true. Also, naming should be simple, like for other bean validation annotations, neither of them have `JavaField` in their name because it's defined show annotation is configured. 

After I wrote this, I realized that we have `jakarta.validation.constraints.Pattern`, so no need in custom annotation that uses regexp at all :-)

Reply:
fixed